serverlessworkflow/modules/ROOT/pages/testing-and-troubleshooting/kn-plugin-workflow-overview.adoc

```diff
@@ -114,27 +117,128 @@ kn workflow create --name my-project
 ----
 --
 
-. Add more extensions to the Quarkus project during its creation by using the `[-e|--extension]` flag as follows:
+[[proc-build-sw-project-kn-cli]]
+== Running a workflow project using Knative CLI
+
+After creating your workflow project, you can use the `run` command with `kn workflow` to build & run your workflow project in your current directory. 
+
+This will start a {product_name} docker image and map your local folder to this image.
+
+.Prerequisites
+* {product_name} plug-in for Knative CLI is installed.
++
+For more information about installing the plug-in, see <<proc-install-sw-plugin-kn-cli, Installing the {product_name} plug-in for Knative CLI>>.
+
+* A workflow project is created.
++
+For more information about creating a workflow project, see <<proc-create-sw-project-kn-cli, Creating workflow project using Knative CLI>>.
+* Minikube cluster is running locally.
+
+
+.Procedure
+. In Knative CLI, enter the following command to build and run your workflow project:
 +
 --
-.Create a project with `quarkus-jsonp` and `quarkus-smallrye-openapi` extensions
+.Run the project and start a local development image.
 [source,shell]
 ----
-kn workflow create --extension quarkus-jsonp,quarkus-smallrye-openapi
+kn workflow run
 ----
+--
+. Navigate to `localhost:8080/q/dev` to access the Development UI locally.
 
-You can add multiple extensions using the comma-separated names of the extensions in the previous command.
+[[proc-deploy-sw-project-kn-cli]]
+== Deploying a workflow project using Knative CLI
+
+You can use the `deploy` command combined with `kn workflow` to deploy your workflow project in your current directory. 
+
+.Prerequisites
+* {product_name} plug-in for Knative CLI is installed.
++
+For more information about installing the plug-in, see <<proc-install-sw-plugin-kn-cli, Installing the {product_name} plug-in for Knative CLI>>.
+
+* A workflow project is created.
++
+For more information about created a workflow project, see <<proc-create-sw-project-kn-cli, Creating workflow project using Knative CLI>>.
+
+* A minikube cluster is running locally.
+
+.Procedure
+. In Knative CLI, enter the following command to deploy your workflow project:
++
+--
+.Deploy a workflow project
+[source,shell]
+----
+kn workflow deploy
+----
+
+.Deploy a workflow project using `--path`
+[source,shell]
+----
+kn workflow deploy --path other-path
```

Comment:
The `--path` flag is only within `quarkus deploy`. The `deploy` takes another flags:
```
Flags:
  -h, --help                        help for deploy
  -c, --manifestPath string         Target directory of your generated Kubernetes manifests.
  -n, --namespace string            Target namespace of your deployment.
  -s, --supportFilesFolder string   Specify a custom support files folder
```

Reply:
And maybe keep the `[-n|--name]` system of mentioning flags that is used later bellow 😉

Reply:
Thanks fixing

Reply:
Ok I removed this, and added mention of help. We can improve with examples for each command in follow-up PR